tests/unittests/test_net.py

```diff
@@ -3537,15 +3537,15 @@
 
                 [ipv4]
                 method=manual
-                may-fail=false
+                may-fail=true
```

Comment:
This is just one example, but is this really the change you want? Brett's proposal was to set `may-fail=true` when dhcp4 and dhcp6 are both set. This accomplishes that, but also will set `may-fail=true` anytime any configuration contains both ipv4 and ipv6 sections regardless of dhcp status.

I just want to double check that you're ok with this broader change.

Reply:
yes you are right. We need to be as narrow as possible with our fix so that we do not break anything else. I have sent a re-work of the patch. Please let me know how it looks.